CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+## [v1.0.14](https://github.com/stargate/data-api/tree/v1.0.14) (2024-07-15)
+
+[Full Changelog](https://github.com/stargate/data-api/compare/v1.0.13...v1.0.14)
+
+**Highlights:**
+
+- Support AWS Bedrock Embedding Provider
+- Set lower default for Azure OpenAI model "text-embedding-3-large" to observed legal value
+- Content updates for form field displayName, hint, and help text
```

Comment:
could we rephrase this to somehow mention that these properties are added to the embedding providers please?

Reply:
Rephrase to "Add `displagName` and `hint` in embedding providers parameters configuration". WDYT?

Reply:
`displagName` typo ?